docs/examples/rx/10_upgrade_chain/index.js

```diff
@@ -0,0 +1,59 @@
+// Import the API & Provider and some utility functions
+const { ApiRx } = require('@polkadot/api');
+const { WsProvider } = require('@polkadot/rpc-provider');
+// import the test keyring (already has dev keys for Alice, Bob, Charlie, Eve & Ferdie)
+const testKeyring = require('@polkadot/keyring/testing');
+const fs = require('fs');
+
+async function main () {
+  // Initialise the provider to connect to the local node
+  const provider = new WsProvider('ws://127.0.0.1:9944');
+
+  // Create the API and wait until ready (optional provider passed through)
+  const api = await ApiRx.create(provider).toPromise();
+
+  // retrieve the upgrade key from the chain state
+  const adminId = await api.query.sudo.key().toPromise();
```

Comment:
Technically, we probably want to switchMap this one. However as it stands, I'm happy with where we are - we can tweak and make it more rx-y along the way. (Same with the create above.)